api/EventTarget.json

```diff
@@ -0,0 +1,507 @@
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "1"
+          },
+          "firefox_android": {
+            "version_added": "1"
+          },
+          "ie": {
+            "version_added": "9"
+          },
+          "opera": {
+            "version_added": "7"
+          },
+          "opera_android": {
+            "version_added": "7"
+          },
+          "safari": [
+            {
+              "version_added": "10.1"
+            }
+            {
+              "version_added": "1",
+              "version_removed": "10.1",
+              "notes": "<code>Window.EventTarget</code> did not exist on versions of Safari before 10.1."
+            }
+          ],
+          "safari_ios": [
+            {
+              "version_added": "10.1"
+            }
+            {
+              "version_added": "1",
+              "version_removed": "10.1",
+              "notes": "<code>Window.EventTarget</code> did not exist on versions of Safari iOS before 10.1."
+            }
+          ]
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "EventTarget": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/EventTarget/EventTarget",
+          "description": "<code>EventTarget()</code> constructor",
+          "support": {
+            "webview_android": {
+              "version_added": "64"
+            },
+            "chrome": {
+              "version_added": "64"
+            },
+            "chrome_android": {
+              "version_added": "64"
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "59"
+            },
+            "firefox_android": {
+              "version_added": "59"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "51"
+            },
+            "opera_android": {
+              "version_added": "51"
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "addEventListener": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/EventTarget/addEventListener",
+          "support": {
+            "webview_android": {
+              "version_added": "1"
+            },
+            "chrome": {
```

Comment:
Chrome has a note here: "Before Chrome 49, the `type` and `listener` parameters were optional."

Reply:
Added